### ISSUE TITLE
Add informations about refresh token or add refresh_csrf_header ?

### ISSUE THREAD

**Issue by johaven** [vimalloc/flask-jwt-extended#37]
```python
def _decode_jwt_from_cookies(type):
    if type == 'access':
        cookie_key = get_access_cookie_name()
    else:
        cookie_key = get_refresh_cookie_name()
    .......
    if get_cookie_csrf_protect() and request.method in get_csrf_request_methods():
        csrf_header_key = get_csrf_header_name()
        ....
```
When you renew token (with a refresh token), you must (on frontend) replace in headers the CSRF content by the content of the REFRESH-CSRF cookie, because only csrf header name can be specified (code above)
It would no be better to create an option to define a refresh_csrf_header or specify this in documentation ?
By default refresh_csrf_header_name could be equal to csrf_header_name, it's an idea.

**Comment by vimalloc** [comment]
Sure, that makes sense. I've had a surprisingly busy week this week, but it seems to be calming down some. I'll try to get some work done on this and the other issues you have opened this weeked. Cheers.

**Comment by vimalloc** [comment]
Released in 1.4.0
